Azure/sonic-buildimage
# Convert docker-nat to buster

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>

**- Why I did it**
This is an enhancement.

**- How I did it**
Remove docker-nat from list of stretch dockers
move iptables to 1.8.2-4 (version in buster)
Rework the iptables patch

**- How to verify it**
Build `target/docker-nat.gz`
Verify that `/etc/apt/sources.list` points to `buster` using `docker exec nat cat /etc/apt/sources.list`

**- Description for the changelog**
Convert docker-nat to buster

**- A picture of a cute animal (not mandatory but encouraged)**
